interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -59,6 +59,9 @@
 // default values for $this_message    
     $this_message = array('message'=>'','message_priority'=>3,'dueDate'=>'');
     $forwarding = false;
+
+// default values for Max words to input in a reminder
```

Comment:
It is not better , it just allowing future changes to the word counter and text area more easily 
Would you like me to add the <textarea maxlength="160" .. ?